app/services/trainees/create_from_dttp.rb

```diff
@@ -172,30 +172,27 @@ def nationality_names
     end
 
     def disability_attributes
-      if disability.blank?
+      if disability.blank? || disability == Diversities::NOT_PROVIDED
```

Comment:
Before, if the disability was recorded as "Not known" in DTTP, this would have fallen to the bottom of this `disability_attributes` method and recorded `disability_disclosure: "disabled"` (line 196) but with no disabilities. Hence the section would be incomplete.

Now, it will fill in the disclosure as not provided, completing the disability section.